gt.js

```diff
@@ -11,19 +11,24 @@ const __ = require('./_internal/placeholder')
  *
  * @synopsis
  * ```coffeescript [specscript]
- * var value any,
- *   leftCompare any,
- *   rightCompare any,
- *   left (value=>Promise|leftCompare)|leftCompare,
- *   right (value=>Promise|rightCompare)|rightCompare
- *
- * gt(left, right)(value) -> Promise|boolean
+ * gt(leftValue any, rightValue any) -> boolean
+ * gt(leftValue any, right function)(value any) -> Promise|boolean
+ * gt(left function, rightValue any)(value any) -> Promise|boolean
+ * gt(left function, right function)(value any) -> Promise|boolean
  * ```
  *
  * @description
  * Test if a left value is greater than (`>`) a right value. Either parameter may be an actual value.
  *
```

Comment:
let's add here

If both arguments are values, `gt` eagerly computes and returns a boolean value.